lib/api/controllers/serverController.js

```diff
@@ -100,6 +104,25 @@ class ServerController {
   /**
    * @returns {Promise<Object>}
    */
+  healthCheck(request) {
+    return this.kuzzle.services.list.memoryStorage.getInfos()
+      .then(() => this.kuzzle.services.list.internalCache.getInfos())
+      .then(() => this.kuzzle.services.list.storageEngine.getInfos())
+      .then(response => {
+        if (['yellow', 'green'].indexOf(response.status) < 0) {
+          throw new ExternalServiceError('Elasticsearch is down: ' + JSON.stringify(response));
+        }
+        return Bluebird.resolve({status: 'green'});
+      })
+      .catch(error => {
+        request.setError(new ServiceUnavailableError(error));
```

Comment:
set here a `ServiceUnavailableError` to force a `503` error code (`ExternalServiceError` code is `500`)